synapseCompare_meta.py

```diff
@@ -39,30 +40,33 @@ def getAllSynapseFiles(projectId):
     allSynapseFiles = getAllSynapseFiles(args.project)
 
     for basename in basename_list:
-        logging.info("Checking: %s" % (basename))
-        basename_platform_alias = tcgaImport.get_basename_platform(basename)
-        logging.info(basename_platform_alias)
-        conf = tcgaImport.getBaseBuildConf(basename, basename_platform_alias, "./")
-        request = conf.buildRequest()
+        try:
```

Comment:
Is this still necessary.  I would remove for now.  It still would miss the current basename when you are banned.